test/__snapshots__/index.spec.js.snap

```diff
@@ -757,8 +646,8 @@ Array [
   Object {
     "force": false,
     "from": "/_next/data/build-id/en/getStaticProps/:id.json",
-    "status": 200,
-    "to": "/.netlify/builders/___netlify-odb-handler",
+    "status": 404,
+    "to": "/server/pages/en/404.html",
```

Comment:
This was introduced in 2aa02dbe8140b2107c312e757e9dc1609c3b4e70. Presumably this is as expected, @orinokai?